src/styles/blocks/card.scss

```
@@ -0,0 +1,111 @@
+@import "../utiles/extends";
+@import "../utiles/resets";
+@import "../blocks/stars.scss";
+@import "../utiles/variables";
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 50px 40px;
+  margin: 0 auto;
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    width: 944px;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+}
+
+.card {
+  display: flex;
+  max-width: 200px;
+  flex-direction: column;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  margin: 0 48px 46px 0;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__picture {
+    margin-top: 32px;
+    @extend %justify-center;
+
+    &-sizes {
+      @extend %justify-center;
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__name {
+    margin-top: 40px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $black-color;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $silver-color;
+  }
+
+  &__star-container {
+    @extend %space-between;
+    margin-top: 16px;
+    height: 16px;
+    font-weight: 400;
+    font-size: 10px;
+  }
+
+  &__price {
+    margin-top: 24px;
+    @extend %space-between;
+
+    &-text {
+      color: $silver-color;
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__buy-link {
+    margin: 16px 0;
+    @extend %justify-center;
+    align-items: center;
+    height: 40px;
+    background: $blue-color;
+    border-radius: 5px;
+
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: $white-color;
+
+    &:hover {
+      background: $white-color;
+      border: 1px solid $blue-color;
```

Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover

```suggestion
      border-color: $blue-color;
```